packages/common/e2e/catalog-collection-qa.e2e.ts

```diff
@@ -3,7 +3,7 @@ import { Collection } from "../src/search/Collection";
 import Artifactory from "./helpers/Artifactory";
 import config from "./helpers/config";
 
-describe("catalog and collection e2e:", () => {
+fdescribe("catalog and collection e2e:", () => {
```

Comment:
I think we want to revert this change, right?

Reply:
it's just in an e2e dev't harness I use to verify that stuff works against the actual api, but I'll remove it